res/qml/main.qml

```diff
@@ -90,132 +88,152 @@ ApplicationWindow {
             height: 120
             visible: root.show4decks && !root.maximizeLibrary
+            width: root.width
 
-            FadeBehavior on visible {
+            FadeBehavior on visible  {
                 fadeTarget: deck4waveform
             }
         }
-
         Skin.DeckRow {
             id: decks12
-
             leftDeckGroup: "[Channel1]"
+            minimized: root.maximizeLibrary
             rightDeckGroup: "[Channel2]"
             width: parent.width
-            minimized: root.maximizeLibrary
         }
-
         Skin.CrossfaderRow {
             id: crossfader
-
             crossfaderWidth: decks12.mixer.width
-            width: parent.width
             visible: !root.maximizeLibrary
+            width: parent.width
 
-            Skin.FadeBehavior on visible {
+            Skin.FadeBehavior on visible  {
                 fadeTarget: crossfader
             }
         }
-
         Skin.DeckRow {
             id: decks34
-
             leftDeckGroup: "[Channel3]"
-            rightDeckGroup: "[Channel4]"
-            width: parent.width
             minimized: root.maximizeLibrary
+            rightDeckGroup: "[Channel4]"
             visible: root.show4decks
+            width: parent.width
 
-            Skin.FadeBehavior on visible {
+            Skin.FadeBehavior on visible  {
                 fadeTarget: decks34
             }
         }
-
         Skin.SamplerRow {
             id: samplers
-
-            width: parent.width
             visible: root.showSamplers
+            width: parent.width
 
-            Skin.FadeBehavior on visible {
+            Skin.FadeBehavior on visible  {
                 fadeTarget: samplers
             }
         }
-
         Skin.EffectRow {
             id: effects
-
-            width: parent.width
             visible: root.showEffects
+            width: parent.width
 
-            Skin.FadeBehavior on visible {
+            Skin.FadeBehavior on visible  {
                 fadeTarget: effects
             }
         }
-
         Skin.Library {
-            width: parent.width
             height: parent.height - y
+            width: parent.width
         }
-
-        move: Transition {
-            NumberAnimation {
-                properties: "x,y"
-                duration: 150
+    }
+    Skin.Settings {
+        id: settingsPopup
+        height: Math.max(840, parent.height * 0.7)
+        modal: true
+        width: Math.max(1400, parent.width * 0.8)
+        x: Math.round((parent.width - width) / 2)
+        y: Math.round((parent.height - height) / 2)
+
+        Overlay.modal: Rectangle {
+            id: overlayModal
+            property real radius: 12
+
+            readonly property bool hasHardwareAcceleration: Mixxx.Config.useAcceleration()
+
+            anchors.fill: parent
+            color: Qt.alpha('#00000010', hasHardwareAcceleration ? 1.0 : 0.6)
+
+            Repeater {
+                model: hasHardwareAcceleration ? 1 : 0
```

Comment:
@Swiftb0y this is where that setting is used. Without this switch, the Windows VM I used to test (which has software acceleration) will max out CPU when opening the popup.
Now I don't want to degrade the look of the new interface just to accommodate unorthodox setup (e.g testing VM) but going forward, we will need to support weaker setups if that makes sense